Cargo.toml

```diff
@@ -45,6 +46,8 @@ async-trait = { version = "0.1.74", default-features = false }
 bech32 = "0.9.1"
 bytes = { version = "1.5.0", default-features = false }
 chrono = "0.4.31"
+clap = { version = "4.5.3", features = ["derive"] }
+color-eyre = "0.6.2"
```

Comment:
Can we confine these deps to the local Cargo.toml since they are only used by the new script?